source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/sensors/stereolabs.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) 2022-2024, The Isaac Lab Project Developers.
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""Configuration for Sterolabs Depth Cameras.
+
+The following configuration parameters are available:
+
+* :obj:`ZED_X_NARROW_RAYCASTER_CFG`: The ZED-X Camera with a narrow field of view as an instance of :class:`RayCasterCameraCfg`.
+* :obj:`ZED_X_MINI_WIDE_RAYCASTER_CFG`: The ZED-X Mini Camera with a wide field of view as an instance of :class:`RayCasterCameraCfg`.
+* :obj:`ZED_X_NARROW_USD_CFG`: The ZED-X Camera with a narrow field of view as an instance of :class:`CameraCfg`.
+* :obj:`ZED_X_MINI_WIDE_USD_CFG`: The ZED-X Mini Camera with a wide field of view as an instance of :class:`CameraCfg`.
+
+Reference:
+
+* https://store.stereolabs.com/cdn/shop/files/ZED_X_and_ZED_X_Mini_Datasheet.pdf?v=10269370652353745680
```

Comment:
This link does not work for me. Perhaps:
https://www.stereolabs.com/store/products/zed-x-stereo-camera -> Resources -> DataSheet
https://www.stereolabs.com/store/products/zed-x-mini-stereo-camera -> Resources -> DataSheet